c7452945.lua

```diff
@@ -6,7 +6,7 @@ function c7452945.initial_effect(c)
 	e1:SetCategory(CATEGORY_EQUIP)
 	e1:SetType(EFFECT_TYPE_ACTIVATE)
 	e1:SetCode(EVENT_FREE_CHAIN)
-	e1:SetProperty(EFFECT_FLAG_CARD_TARGET)
+	e1:SetProperty(EFFECT_FLAG_CARD_TARGET+EFFECT_FLAG_CONTINUOUS_TARGET+EFFECT_FLAG_CONTINUOUS_TARGET)
```

Comment:
`EFFECT_FLAG_CONTINUOUS_TARGET` has twice.

Reply:
fixed
Thank you very much!